src/middlewares/error-logger.js

```diff
@@ -1,14 +1,19 @@
 import logger from '../lib/logger';
+import { RequestValidationError } from '../lib/errors';
+import { flags } from '../config';
 
 // NOTE: It's only stub for logger
 
 // TODO: Should write non custom error in log file.
 //       After that developer use information from log file for writing new errors class.
-const errorLogger = (error, requset, response, next) => {
-  logger.error('↓'.repeat(error.message.length));
-  logger.error(error);
-  logger.error('↑'.repeat(error.message.length));
-  next();
+const errorLogger = (error, request, response, next) => {
+  if (error.name === RequestValidationError.name) {
```

Comment:
Needs better conditions. If the only way it logs errors is when the debug flag is true, then the middleware altogether should be applied conditionally (up the level)
If you need to conditionally not log some errors (don't know why you wouldn't), then use a switch-case here, something like
```javascript
switch(error.name) {
   case blabla.name:
   case RequestValidationError.name:
         next();
    default:  
       logger.error(error);
}
```